README.md

```diff
@@ -4,9 +4,199 @@ CartoDB Spatial Analysis extension for PostgreSQL.
 
 ## Code organization
 
-* *pg* contains the PostgreSQL extension source code
-* *python* Python module
+* *doc* documentation
+* *src* source code
+* - *src/pg* contains the PostgreSQL extension source code
+* - *src/py* Python module source code
+* *release* reseleased versions
+* *env* base directory for Python virtual environments
 
 ## Requirements
 
-* pip
+* pip, virtualenv, PostgreSQL
+* python-scipy system package (see src/py/README.md)
+
+# Working Process
+
+We use the default `develop` branch as the basis for development.
+This branch and `master` are maintained by the *Release Manager*.
+The `master` branch is used to merge and tag releases to be
+deployed in production.
+
+In addition to these two permanent branches, temporal *topic*
+branches must be used for all modifications.
+
+## Development
+
+For any modification of crankshaft, including adding new features,
+a topic branch must be created out of the `develop` branch
+and be used for the development process; see src/py/README.md
+for further details about the Pyhton package development.
```

Comment:
I do believe how branches are managed is independent of whether the changes lie in sql or python code, right?